bin/cli.js

```diff
@@ -222,6 +248,29 @@ function invoke(env) {
         .catch(exit);
     });
 
+  commander
+    .command('migrate:down')
+    .description('        Rollback the last migration performed.')
```

Comment:
Description seems inaccurate.

Reply:
What are your thoughts on `Undo the last previous migration performed.`?

Reply:
I'm more concerned about 
a) usage of --to flag making description completely incorrect;
b) difference between last migration and last migration batch being not emphasized enough;

Reply:
Oh I see what you mean. That makes complete sense! When working on a pull request for this I'll try and think of a more accurate and all encompassing description 👍